internal/command/generate.go

```diff
@@ -71,3 +85,27 @@ var CmdGenerate = &Command{
 		return nil
 	},
 }
+
+// Checks if the library with the given API path exists in the remote pipeline state
+func checkIfLibraryExistsInLanguageRepo(ctx *CommandContext) string {
+	if flagRepoUrl == "" {
```

Comment:
I think we should probably handle repo-root as well, but that could come later. Maybe file an issue and I'll look at it another time?